.circleci/install.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash -ex
+# -e means:  Exit immediately if a command exits with a non-zero status
+# -x means:  Print commands and their arguments as they are executed.
+
+# Install some required system packages
+sudo apt-get update
+sudo apt-get install libegl1-mesa
+
+# Install Miniconda and required packages using astropy's ci-helpers
+TRAVIS_OS_NAME='linux'
+CONDA_DEPENDENCIES='notebook pytest pytest-cov flaky'
```

Comment:
Please add below this step

    CONDA_DEPENDENCIES_FLAGS="--quiet"

to avoid a lot of unnecessary noise in this step.